plugins/templating/src/main/kotlin/templates/DirectiveBasedTemplateProcessing.kt

```diff
@@ -20,9 +25,17 @@ class DirectiveBasedHtmlTemplateProcessingStrategy(private val context: DokkaCon
         if (input.isFile && input.extension == "html") {
             val document = Jsoup.parse(input, "UTF-8")
             document.outputSettings().indentAmount(0).prettyPrint(false)
+
             document.select("dokka-template-command").forEach {
                 handleCommand(it, parseJson(it.attr("data")), input, output)
             }
+            extractCommandsFromComments(document) { nodes, command ->
+                val nodesTrimed =
+                    nodes.dropWhile { (it is TextNode && it.isBlank).also { res -> if (res) it.remove() } }
+                        .dropLastWhile { (it is TextNode && it.isBlank).also { res -> if (res) it.remove() } }
```

Comment:
> Also, am I correct to understand that we remove all blank text nodes in the beginning and in the end, but leave empty nodes in the middle? Is it just in case or was there something specific?


Because of formatting (indenting) of tags.

```
    <!--[+]cmd:-->
    <link href="###styles/main.css" rel="Stylesheet">
    <link href="###styles/main.css" rel="Stylesheet">
    <!--[-]cmd-->
```
----->
 ```
    <link href="###styles/main.css" rel="Stylesheet">
    <link href="###styles/main.css" rel="Stylesheet">
```